docs/defi/rosetta/icp_rosetta/construction_api/staking.mdx

```diff
@@ -245,6 +245,60 @@ curl --location '0.0.0.0:8081/construction/payloads'  --header 'Content-Type: a
 }'
 ```
 
+## Disburse maturity
+
+|                      |        |
+|----------------------|--------|
+| Since version        | 2.1.8  |
+| Idempotent?          | No    |
+| Minimal access level | controller |
+
+This section will show you how to use Rosetta to disburse maturity associated with a neuron.
```

Comment:
nit: I find that direct language is easier to read in documentation, but that's a personal preference. Your call.

```suggestion
Disburses a neuron's maturity to the neuron's controller or a specified recipient.
```